easybuild/easyconfigs/u/UDUNITS/UDUNITS-2.2.28-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg, Ghent University
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (Ghent University)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-97.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'UDUNITS'
+version = '2.2.28'
+
+homepage = 'https://www.unidata.ucar.edu/software/udunits/'
+description = """UDUNITS supports conversion of unit specifications between formatted and binary forms,
+ arithmetic manipulation of units, and conversion of values between compatible scales of measurement."""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://artifacts.unidata.ucar.edu/repository/downloads-udunits/%(version)s/',
+    'https://sources.easybuild.io/u/UDUNITS/',
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['590baec83161a3fd62c00efa66f6113cec8a7c461e3f61a5182167e0cc5d579e']
+
+builddependencies = [('binutils', '2.42')]
```

Comment:
I see configure checking for `flex`, `bison` and `libtool`, which we should probably provide as build deps